uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -640,83 +727,27 @@ define([
       });
     }
 
-    /**
-     * This should be called whenever a date has been changed
-     * First it syncs `from` and `to` date, if it's in 'single day' mode
-     * Then, if all the dates are there, it gets the balance change
-     *
-     * @param {Date} date - the selected date
-     * @param {String} dayType - set to from if from date is selected else to
-     * @return {Promise}
-     */
-    function updateAbsencePeriodDatesTypes (date, dayType) {
-      return vm.loadAbsencePeriodDatesTypes(date, dayType)
-        .then(function () {
-          return vm.updateBalance();
-        })
-        .catch(function (errors) {
-          handleError(errors);
-          vm._setDateAndTypes();
-          vm._toggleBalance();
-        });
-    }
-
     /**
      * Whenever the absence type changes, update the balance opening.
```

Comment:
it seems that this documentation has nothing to do with this function

Reply:
I noticed there was some duplicating logic so I completely separated "openingbalance" and "closingbalance" calculations.